ledger/sandbox-on-x/reference.conf

```
@@ -107,6 +107,10 @@ ledger {
     # participant instance defined by participant-id. Defined as `participant-0` here.
     participant-0 {
 
+      # An override of `participant-id` as defined in the configuration map.
```

The below might be easier to understand:
```suggestion
      # An override of `participant-id` as defined by the configuration key above.
```